docs/protocol/tokenomics.md

```diff
@@ -0,0 +1,98 @@
+The Autonolas Protocol tokenomics is the economic model designed to foster the development and composability of autonomous services. The primary goal is to create a sustainable ecosystem to incentivize developers to contribute to the network and reward them for their participation proportionally to their efforts, while the platform itself grows and becomes more valuable.
+
+Autonolas uses a **bonding mechanism** to grow the protocol’s capital, a **staking model** for code to enable developers to track their code contributions on-chain, and **a system to receive rewards** for code usefulness. Additionally, Autonolas aims to create a flywheel to **attract donations** and provide decentralized protocol-owned services to ensure ecosystem longevity.
+
+!!! abstract "Learn more"
+
+    Read the **Tokenomics** section in the [Autonolas Whitepaper](https://www.autonolas.network/documents/whitepaper/Whitepaper%20v1.0.pdf) for the full details.
```

Comment:
```suggestion
    Read the **Tokenomics** section in the [Autonolas Whitepaper](https://www.autonolas.network/documents/whitepaper/Whitepaper%20v1.0.pdf) for a further overview.
```

Reply:
I understand not all the details might be in the Whitepaper, but the rest of warnings in the doc are framed in this way.